Modules/IO/DCMTK/test/itkDCMTKGetDicomTagsTest.cxx

```diff
@@ -0,0 +1,157 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkImageFileReader.h"
+#include "itkImageFileWriter.h"
+#include "itkDCMTKFileReader.h"
+#include "itkTestingMacros.h"
+
+#include <fstream>
+
+int
+itkDCMTKElementGetTest(int ac, char * av[])
+{
+
+  if (ac != 2)
+  {
+    std::cerr << "Missing Parameters." << std::endl;
+    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(av) << " DicomImage" << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  constexpr unsigned int Dimension = 3;
+  using InputPixelType = unsigned short;
+  using InputImageType = itk::Image<InputPixelType, Dimension>;
+  using ReaderType = itk::ImageFileReader<InputImageType>;
+
+  itk::DCMTKFileReader fileReader;
+  fileReader.SetFileName(av[1]);
+  fileReader.LoadFile();
+
+  bool throwException = false;
+
+  std::string strTarget;
+
+  // Study Date
+  unsigned short group = 0x0008;
+  unsigned short element = 0x0021;
+  if (fileReader.GetElementDA(group, element, strTarget, throwException) == EXIT_SUCCESS)
+  {
+    // YYYYMMDD
+    // Expect 20030625
+    std::cout << strTarget << std::endl;
+  }
+
+  // Study Time
+  group = 0x0008;
+  element = 0x0030;
+  // seq.GetElementTM(group, element, tag)
+  if (fileReader.GetElementTM(group, element, strTarget, throwException) == EXIT_SUCCESS)
+  {
+    // Expect 152734
+    std::cout << strTarget << std::endl;
+  }
+
+  // Patient Name
+  // Expect "Wes Turner"
+  group = 0x0010;
+  element = 0x0010;
+  if (fileReader.GetElementPN(group, element, strTarget, throwException) == EXIT_SUCCESS)
+  {
+    std::cout << strTarget << std::endl;
+  }
+
+  // Patient Sex
+  // Expect "O"
+  group = 0x0010;
+  element = 0x0040;
+  if (fileReader.GetElementCS(group, element, strTarget, throwException) == EXIT_SUCCESS)
+  {
+    std::cout << strTarget << std::endl;
+  }
+
+  // Patient Weight
+  // Expect 68.039
+  group = 0x0010;
+  element = 0x1030;
+  if (fileReader.GetElementDS(group, element, 1, &strTarget, throwException) == EXIT_SUCCESS)
```

Comment:
&strTarget withput &

Reply:
Change requested (s. above comment).

Reply:
:100: thanks for the review @issakomi. Done. Push-forced in 73c450e.